dev/storage.h

```diff
@@ -1027,9 +1139,12 @@ namespace sqlite_orm {
              */
             bool table_exists(const std::string& tableName) {
                 auto con = this->get_connection();
-                return this->impl.table_exists(tableName, con.get());
+                return this->table_exists(tableName, con.get());
             }
 
+            // jdh
```

Comment:
please remove such comments. We use git especially for this - to know who wrote any line

Reply:
go